### ISSUE TITLE
Adding a show with a colon in name fails

### ISSUE THREAD

**Issue by G33K** [pymedusa/Medusa#8246]
**Describe the bug**
When adding a show with a colon in the name for example "Space: 1999", the show is not added. The folder is created correctly but without the colon in the folder name and with the artwork downloaded in to it. However the show does not get added to the database and does not show up in any lists. 

**To Reproduce**
Try adding a show with a colon in its name using the TVDBv2 indexer

**Medusa (please complete the following information):**
- OS Debian 9.12
- Branch: master
- Commit: d9c31da063e6c4b4383749e19b27b13693a1b71e
- Version: 0.3.16
- Database: 44.15
- Python Version: | 3.5.3

**Comment by p0psicles** [comment]
Works fine here.
Pls enable debug logs.
restart medusa
add show
post debug logs of the SHOWQUEUE-ADD thread.

![image](https://user-images.githubusercontent.com/1331394/87970607-45b34a00-cac4-11ea-91cf-a38b0803661b.png)

**Comment by G33K** [comment]
Looks like this might be a local issue on my end. I also just tested it on a totally fresh install on a different server and it added successfully but I can't get it to add on this server.
Looking at the log I'm seeing it hang at fetching the info from IMDB because there is no imdb id. I have tried restarting Medusa, restarting the server, clearing the cache folder but looks like it is still pulling erroneously cached info from somewhere. Where else is the info from TVDB cached?
Surprisingly, I don't have an indexer folder in my cache which is stated in the logs

`
2020-07-20 13:07:29 DEBUG	SHOWQUEUE-ADD :: [d9c31da] 76366: Loading show info from IMDb with ID: https://www.imdb.com/title/
2020-07-20 13:07:29 DEBUG	SHOWQUEUE-ADD :: [d9c31da] Retrieving show info from IMDb
2020-07-20 13:07:29 INFO	SHOWQUEUE-ADD :: [d9c31da] Setting all previously aired episodes to the specified status: Skipped
2020-07-20 13:07:29 DEBUG	SHOWQUEUE-ADD :: [d9c31da] TVDBv2 76366: Setting location: /mnt/TV/TV Series/Space 1999
2020-07-20 13:07:29 DEBUG	SHOWQUEUE-ADD :: [d9c31da] Transaction with 2 queries executed
2020-07-20 13:07:29 DEBUG	SHOWQUEUE-ADD :: [d9c31da] Trying other indexer: TMDB get_id_by_external
2020-07-20 13:07:28 DEBUG	SHOWQUEUE-ADD :: [d9c31da] Could not get tvmaze externals using external key tvdb_id and id 76366
2020-07-20 13:07:28 DEBUG	SHOWQUEUE-ADD :: [d9c31da] User-Agent: Medusa/0.3.16 (Linux; 4.9.0-12-amd64; 7639c2f8-cac4-11ea-81a1-002590af120d)
2020-07-20 13:07:28 DEBUG	SHOWQUEUE-ADD :: [d9c31da] GET URL: http://api.tvmaze.com/lookup/shows?thetvdb=76366 [Status: 404]
2020-07-20 13:07:28 DEBUG	SHOWQUEUE-ADD :: [d9c31da] Could not get tvmaze externals using external key imdb_id and id https://www.imdb.com/title/
2020-07-20 13:07:28 DEBUG	SHOWQUEUE-ADD :: [d9c31da] User-Agent: Medusa/0.3.16 (Linux; 4.9.0-12-amd64; 7639c2f8-cac4-11ea-81a1-002590af120d)
2020-07-20 13:07:28 DEBUG	SHOWQUEUE-ADD :: [d9c31da] GET URL: http://api.tvmaze.com/lookup/shows?imdb=https://www.imdb.com/title/ [Status: 404]
2020-07-20 13:07:28 DEBUG	SHOWQUEUE-ADD :: [d9c31da] Trying other indexer: TVmaze get_id_by_external
2020-07-20 13:07:28 DEBUG	SHOWQUEUE-ADD :: [d9c31da] 76366: Loading show info from TVDBv2
2020-07-20 13:07:28 INFO	SHOWQUEUE-ADD :: [d9c31da] 76366: Unable to find the show in the database
2020-07-20 13:07:28 DEBUG	SHOWQUEUE-ADD :: [d9c31da] 76366: Loading show info from database
2020-07-20 13:07:28 DEBUG	SHOWQUEUE-ADD :: [d9c31da] Trying other indexer: TMDB get_id_by_external
2020-07-20 13:07:27 DEBUG	SHOWQUEUE-ADD :: [d9c31da] Could not get tvmaze externals using external key imdb_id and id https://www.imdb.com/title/
2020-07-20 13:07:27 DEBUG	SHOWQUEUE-ADD :: [d9c31da] User-Agent: Medusa/0.3.16 (Linux; 4.9.0-12-amd64; 7639c2f8-cac4-11ea-81a1-002590af120d)
2020-07-20 13:07:27 DEBUG	SHOWQUEUE-ADD :: [d9c31da] GET URL: http://api.tvmaze.com/lookup/shows?imdb=https://www.imdb.com/title/ [Status: 404]
2020-07-20 13:07:27 DEBUG	SHOWQUEUE-ADD :: [d9c31da] Trying other indexer: TVmaze get_id_by_external
2020-07-20 13:07:27 DEBUG	SHOWQUEUE-ADD :: [d9c31da] User-Agent: Medusa/0.3.16 (Linux; 4.9.0-12-amd64; 7639c2f8-cac4-11ea-81a1-002590af120d)
2020-07-20 13:07:27 DEBUG	SHOWQUEUE-ADD :: [d9c31da] GET URL: https://api.thetvdb.com/series/76366/episodes/query?page=1 [Status: 200]
2020-07-20 13:07:27 DEBUG	SHOWQUEUE-ADD :: [d9c31da] Getting all episodes of 76366
2020-07-20 13:07:27 DEBUG	SHOWQUEUE-ADD :: [d9c31da] User-Agent: Medusa/0.3.16 (Linux; 4.9.0-12-amd64; 7639c2f8-cac4-11ea-81a1-002590af120d)
2020-07-20 13:07:27 DEBUG	SHOWQUEUE-ADD :: [d9c31da] GET URL: https://api.thetvdb.com/series/76366 [Status: 200]
2020-07-20 13:07:27 DEBUG	SHOWQUEUE-ADD :: [d9c31da] Getting all show data for 76366
2020-07-20 13:07:27 DEBUG	SHOWQUEUE-ADD :: [d9c31da] Getting all series data for 76366
2020-07-20 13:07:27 DEBUG	SHOWQUEUE-ADD :: [d9c31da] Configured language en override show language of en
2020-07-20 13:07:27 INFO	SHOWQUEUE-ADD :: [d9c31da] TVDBv2: {'use_zip': True, 'cache': '/var/medusa/cache/indexers/TVDBv2', 'language': 'en', 'session': <medusa.session.core.MedusaSession object at 0x7ff69c1a0630>}
2020-07-20 13:07:27 INFO	SHOWQUEUE-ADD :: [d9c31da] Starting to add show by show_dir: /mnt/TV/TV Series/Space 1999
`

**Comment by G33K** [comment]
So, after waiting a few days for the cache to expire I was able to successfully add the series to the database. As suspected, due to the erroneous cached data the add process was crashing. Now with a fresh grab of the data as seen in the logs it added successfully.

This can be closed but I would still like to know where Medusa caches the TVDB data it grabs since I do not see an indexers folder in the cache folder as seen in the logs and I did restart the Medusa and the server and removed the cache folder along with the cache.db files to no avail.
